bears/python/PyreBear.py

```diff
@@ -0,0 +1,29 @@
+import os
+
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.PipRequirement import PipRequirement
+
+
+@linter(executable='pyre',
+        output_format='regex',
+        normalize_column_numbers=True,
+        output_regex=r'\s*(?P<filename>.+)\s*:(?P<line>\d+):(?P<column>\d+)'
+                     r' \s*(?P<message>.+)',
+        global_bear=True)
+class PyreBear:
+    """
+    Checks Python files for errors using ``pyre``.
+    See https://pyre-check.org/ for more info.
+    """
+    LANGUAGES = {'Python', 'Python 3'}
+    REQUIREMENTS = {PipRequirement('pyre-check', '0.0.12')}
```

Comment:
CI is now complaining with `Could not find a version that satisfies the requirement pyre-check~=0.0.12`

Reply:
Do you mean `appveyor` ? 
It's running `python3.4` and as mentioned above, pyre-check is only available `python3.5` or above, that's why it's failing.